Index: debhello/deploy/packaging_deb.sh
===================================================================
--- debhello.orig/deploy/packaging_deb.sh
+++ debhello/deploy/packaging_deb.sh
@@ -52,11 +52,11 @@ tar -zxvf ${PACKAGE_DIR_NAME}.tar.gz -C
 pushd ${PACKAGE_DIR_NAME}/
 
 debmake
+quilt push -a
 
 cp ${DEB_OVERWRITE}/debian/rules     debian/rules
 cp ${DEB_OVERWRITE}/debian/control   debian/control
-cp ${DEB_OVERWRITE}/debian/copyright debian/copyright
-
+#cp ${DEB_OVERWRITE}/debian/copyright debian/copyright
 
 # build .deb file
 debuild -us -uc
